openslides/global_settings.py

```diff
@@ -11,7 +11,7 @@
 
 AUTHENTICATION_BACKENDS = ('openslides.users.auth.CustomizedModelBackend',)
 
-LOGIN_URL = '/login/'
+LOGIN_URL = '/users/'
```

Comment:
At the moment '/' is not handled by angular. '/users/' is the most basic url there is.